build/stackmat.js

```diff
@@ -317,7 +317,11 @@
       this.start = bind(this.start, this);
       this.signalFetched = bind(this.signalFetched, this);
       if (!supported()) {
-        alert("You need a recent browser in order to connect your Stackmat Timer.");
+        if (options != null ? options.onNonSuppotedBrowser : void 0) {
```

Comment:
oops!

In fact, now it's not necessary useful to add this file in the repository because this file is always rebuild when you publish on npm (via the `prepublish` hook in the `package.json`